vnext/Desktop.IntegrationTests/WebSocketResourcePerformanceTests.cpp

```diff
@@ -45,34 +48,58 @@ return -1;
 /// test must be run in isolation (no other tests running concurrently).
 ///
 TEST_METHOD(ProcessThreadsPerResource) {
-  const int resourceTotal = 50; // About 3 seconds total running time. 6 if we
-  // increase this value to 100.
+  // About 3 seconds total running time.
+  // 6, if we increase this value to 100.
+  const int resourceTotal = 50;
   const int maxWriteCount = 10;
   const int expectedThreadsPerResource = 3;
   const int startThreadCount = GetCurrentThreadCount();
 
   std::atomic_int32_t threadCount = 0;
+  bool errorFound = false;
+  string errorMessage;
+
+  auto server = std::make_shared<Test::WebSocketServer>(5556);
+  server->SetMessageFactory([](string &&message) { return message + "_response"; });
+  // TODO: Re-enable.
+  // server->Start();
 
   // WebSocket resources scope.
   {
     vector<shared_ptr<IWebSocketResource>> resources;
     for (int i = 0; i < resourceTotal; i++) {
-      auto ws = IWebSocketResource::Make("ws://localhost:5556/");
-      ws->SetOnMessage([this, &threadCount](size_t size, const string &message) {
+      auto ws = IWebSocketResource::Make("ws://localhost:5555/"); // TODO: Switch to port 5556
```

Comment:
Why the change here? Does one have TLS or something?

Reply:
I'm curious what's going on here more in general, with having the server running on a previous port and no longer calling Start/Stop on it.

Reply:
No. Currently, the in-proc WebSocket server doesn't support multiple requests (yes, that's pretty limited). I deafult that server to port 5556 so it doesn't collide with the Node one provided by React Native.

For now, I'm using the Node server to run this test.

Reply:
> I'm curious what's going on here more in general, with having the server running on a previous port and no longer calling Start/Stop on it.

It's only declared, but disabled (never runs).
I could as well just remove it and leave a comment to re-introduce it later.